### PR TITLE
tests: increase code coverage at consecutive

### PR DESCRIPTION
**Pull request by qalisander**
Adds missing tests to erc721 consecutive mostly associated with erc721 standard's operations.

### CODE REVIEW

File: contracts/src/token/erc721/extensions/consecutive.rs [hunk 1]
```diff
@@ -994,5 +1034,300 @@ mod tests {
...
+            .safe_transfer_from(BOB, alice, token_id)
+            .expect("should transfer Bob's token to Alice");
+        let owner = contract
+            .owner_of(token_id)
+            .expect("should return the owner of the token");
+        assert_eq!(owner, alice);
+    }
+
+    #[motsu::test]
+    fn error_when_safe_transfer_transfers_from_incorrect_owner(
```
Comment (bidzyyys) [inline contracts/src/token/erc721/extensions/consecutive.rs:1085]: I would skip the naming convention we decided some time ago and leave: `error_when_safe_transfers_from_incorrect_owner`

File: contracts/src/token/erc721/mod.rs [hunk 2]
```diff
@@ -2306,7 +2306,7 @@ mod tests {
     }
 
     #[motsu::test]
-    fn error_when_safe_transfer_internal_ransfers_to_invalid_receiver(
+    fn error_when_safe_transfer_internal_transfers_to_invalid_receiver(
```
Comment (bidzyyys) [inline contracts/src/token/erc721/mod.rs:2309]: Good catch!

**Review by bidzyyys — APPROVED** [review]
Left one comment regarding naming convention.

**Review by ggonzalez94 — APPROVED** [review]
LGTM

**Pull request merged**